modules/timepoint_list/php/timepoint_list.class.inc

```diff
@@ -309,16 +309,14 @@ class Timepoint_List extends \NDB_Menu
         $candid = $this->tpl_data['candID'];
         $pscid  = $this->tpl_data['PSCID'];
 
-        $link = ltrim($_SERVER['REQUEST_URI'], '/');
-
         return new \LORIS\BreadcrumbTrail(
             new \LORIS\Breadcrumb(
                 'Access Profile',
                 '/candidate_list'
             ),
             new \LORIS\Breadcrumb(
                 "Candidate Profile $candid / $pscid",
-                "/$candid"
+                "$candid"
```

Comment:
i see this issue in timepoint list but i don't think this is the solution. I wonder if $candid has a `\` in it when it shouldn't? the fix might be in there. could maybe use `$this->candID` directly.

Reply:
add: i am proposing this because this issue doesn't seem to be appearing anywhere else and   the code for other modules include a `/` in the candid input for `new \LORIS\Breadcrumb`

Reply:
@zaliqarosli okay.